packages/aws-cdk-lib/aws-route53/lib/util.ts

```diff
@@ -69,3 +70,24 @@ export function makeHostedZoneArn(construct: Construct, hostedZoneId: string): s
     resourceName: hostedZoneId,
   });
 }
+
+export function makeGrantDelegation(grantee: iam.IGrantable, hostedZoneArn: string) {
```

Comment:
Please specify the return type in the function definition (looks like iam.Grant or iam.IGrantable)